app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  validates :username, presence: true
```

Comment:
- [ ] **[OPTIONAL]** You can list multiple attributes if they share the same validator =>
```ruby
validates :username, :email, :password, presence: true
```